packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -393,6 +398,12 @@ class _AppBarState extends State<AppBar> {
         crossAxisAlignment: CrossAxisAlignment.stretch,
         children: widget.actions,
       );
+    } else if (hasEndSideDrawer) {
+      actions = new IconButton(
+        icon: const Icon(Icons.menu),
+        onPressed: _handleDrawerButtonEndSide,
+        tooltip: MaterialLocalizations.of(context).openAppDrawerTooltip,
+      );      
```

Comment:
Does the mean the button to open the end-side drawer appears only when there are no other actions in the app bar?

Reply:
Yes, that is the assumption.